xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -189,6 +189,7 @@ struct SelectionStream
   std::string codec;
   std::string codecDesc;
   int channels = 0;
+  std::string m_channelLayout;
```

Comment:
prefix "m_" could be removed its not mandatory on struct

Reply:
actually, although I don't agree with it, this is the guideline https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#86-structs

Reply:
its mandatory pascal case not the prefix
you can read full discussion on original PR code change

Reply:
see the example of the guideline. this is confusing.

Reply:
PascalCase is for the struct name, members are described in https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#86-structs

Reply:
I'm not going back to discuss this, it was discussed extensively in the PR amending doc guidelines
for members mainly refers to the class variables
differently structs allow direct access to variables so simplify reading, if the struct needs to have also methods of some kind the prefixes may be more useful

do as you wish, but then don't point your finger at PRs that don't use prefixes on structs

I pointed out because in particular this struct does not make use of prefixes, so just putting it here is a little inconsistent